src/main/java/core/basesyntax/model/Truck.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.model;
+
+import core.basesyntax.Machine;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Common Truck, You can do it");
```

Comment:
The message inside the 'doWork' method should be more informative and indicate that it is a Truck which is working. For example, 'Truck is starting its work' would be more appropriate.